File: Python Code/Daily_Flash_Code_0/Week 3/1Feb/17-DailyFlash_Solutions/17-DailyFlash_Solutions/01_Feb_Solutions_Two/Python/prog5.py
num1 = int(input("num1 : "))
num2 = int(input("num2 : "))
while(True):
    rem = num1 % num2
    if(rem == 0):
        break
    num1 = num2
    num2 = rem
print("gcd : ", num2)

File: Python Code/Daily_Flash_Code_0/Week 3/1Feb/17-DailyFlash_Solutions/17-DailyFlash_Solutions/01_Feb_Solutions_Two/C++/prog5.cpp
#include<iostream>

int main() {
	int num1, num2, rem = 1;
	std::cout << "num1 : ";
	std::cin >> num1;
	std::cout << "num2 : ";
	std::cin >> num2;
	while(1) {
		rem = num1 % num2;
		if(rem == 0)
			break;
		num1 = num2;
		num2 = rem;
	}
	std::cout << "gcd : " << num2;
	return 0;
}
